src/routes/_components/list/ListItem.html

```diff
@@ -1,4 +1,17 @@
-<div class="list-item"
+<style>
+  .list-item {
+    opacity: 0;
+    pointer-events: none;
+    transition: opacity 0.2s linear;
+    contain: content; /* see https://www.w3.org/TR/2018/CR-css-contain-1-20181108/#valdef-contain-content */
+  }
+
+  .list-item.shown {
+    opacity: 1;
+    pointer-events: auto;
+  }
+</style>
+<div class="list-item {shown && 'shown'}"
```

Comment:
Forgot to make the non-virtual list have a nice fade-in animation, fixed that while I was here